static/js/play_finish.js

```diff
@@ -158,7 +158,13 @@ let app = new Vue({
 
 
         generateResults: function(event) {
-            return `Wiki Speedruns ${this.promptId}\n${this.startArticle}\n${this.path.length - 1} 🖱️\n${(this.playTime)} ⏱️`
+            if(this.isQuickRun){
```

Comment:
I would probably restructure this into something like

```
let resultText = `Wiki Speedruns ${this.promptId}\n${this.startArticle}\n${this.path.length - 1} 🖱️\n${(this.playTime)} ⏱️`

if (this.isQuickRun) {
    link = `...`
     resultText += \n${link}
}

return resultText
```

But also this would be a lot cleaner with a shortlink, thoughts on blocking this until then @dqian3 ?

Reply:
Thanks for the suggestion. Just pushed a new commit.

Reply:
Ehh, I'm fine with having this for now, I'll merge it then. Thanks for the contribution @nightingale999 !